ui/lib/kv/addon/components/kv-version-dropdown.hbs

```diff
@@ -1,7 +1,7 @@
 <BasicDropdown @class="popup-menu" @horizontalPosition="auto-right" @verticalPosition="below" as |D|>
   <D.Trigger data-test-version-dropdown class="toolbar-link {{if D.isOpen ' is-active'}}">
     Version
-    {{@secret.version}}
+    {{or @currentVersion "current"}}
```

Comment:
On various permissions tests I noticed a few cases where the dropdown just said `version` -- figured we can have a fallback to current since that is the default view

Reply:
🤔 Is that from no access to secret? or just not passed in/set up correctly? If they don't have access to read the secret it seems like the dropdown should hide all together

Reply:
I wasn't able to quite track it down but hopefully this is just an edge case. I think not including version when the read fails was part of the issue